easybuild/easyconfigs/r/R/dbarts-0.9-18_fix-simd-on-non-x86.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru dbarts.orig/src/misc/simd.c dbarts/src/misc/simd.c
```

Comment:
@terjekv Please add a short description of the patch on top + mention author or upstream reference.

Reply:
@zao is magic. Fixed.